hrm-domain/lambdas/search-index-consumer/messagesToPayloads.ts

```diff
@@ -40,39 +43,101 @@ export type PayloadsByIndex = {
 };
 export type PayloadsByAccountSid = Record<AccountSID, PayloadsByIndex>;
 
-// TODO: Pull the transcripts from S3 (if any)
+type IntermidiateIndexContactMessage = MessageWithMeta & {
```

Comment:
'intermediate' is slightly vague naming, could we use something more descriptive of what it is, like `IndexContactMessageWithTranscript` or `ContactIndexingInputData` or something? This is beasically all the raw data we need to generate an index document for ES, or more specifically the payload for the API to create the document. Would it be possible to rejig the naming of the types and methods here to clarify that?

Reply:
Yes those could have a better name :P Thanks for the feedback! I'm going with `ContactIndexingInputData` (and alike) just in case we have more extra info to index at some point, other than just transcripts.